.azure/pipelines/helix-matrix.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       displayName: Restore interop projects
     - script: .\eng\build.cmd -ci -nobl -noBuildRepoTasks -NoRestore -test -all -noBuildNative -projects eng\helix\helix.proj
               /p:IsHelixDaily=true /p:IsRequiredCheck=true /p:IsHelixJob=true /p:BuildInteropProjects=true
-              /p:RunTemplateTests=true /p:CrossgenOutput=false /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log
+              /p:RunTemplateTests=false /p:CrossgenOutput=false /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log
```

Comment:
Assuming something else fails and we we need another build, we should probably revert this as its not needed now that we don't build the template tests on helix, so its one less thing to remember to undo in the future

Reply:
Just did this.